_data/contributors.yml

```diff
@@ -4638,7 +4640,7 @@
   packages_reviewed:
   location:
   email:
-- name:
+- name: Beryl Kanali
```

Comment:
@pllim i have a test run that we can use here now !! it captured the changes that you just made to the names!! yay!!